neuraxle/steps/sklearn.py

```diff
@@ -44,6 +45,13 @@ def __init__(
         super().__init__(hyperparams=params, hyperparams_space=hyperparams_space)
         self.return_all_sklearn_default_params_on_get = return_all_sklearn_default_params_on_get
 
+    def fit_transform(self, data_inputs, expected_outputs=None) -> ('BaseStep', Any):
+        self.wrapped_sklearn_predictor = self.wrapped_sklearn_predictor.fit(data_inputs, expected_outputs)
```

Comment:
Also consider `if hasattr(self.wrapped_sklearn_predictor, 'fit_transform'):` which is important to save time (e.g.: avoid doing `fit` then `transform` which might duplicate some computations and can cause pipelines to take 2x the time to compute). Opened issue, you can merge for now.

Reply:
#71

Reply:
not supposed to be in this pr